src/pages/seat-reservation/data.js

```diff
@@ -0,0 +1,96 @@
+const cinemaHalls = [
+        {
+            numberOfSeats: { rows: 9, columns: 14 },
+            filmShowing: [
```

Comment:
jeśli chodzi o senas to wystarczy filmShowingId a konkretny seans weźmiemy z kolekcji od Sylwii

Reply:
Ok, w takim razie poczekam na merge'a taska od Sylwii i to zmienię